Source/EasyTipView.swift

```diff
@@ -139,7 +139,6 @@ public extension EasyTipView {
             self.alpha = self.preferences.animating.dismissFinalAlpha
         }) { (finished) -> Void in
             completion?()
-            self.delegate?.easyTipViewDidDismiss(self)
```

Comment:
`dismiss` is a public method which can be called by anyone, therefore, an implicit delegation call inside the completion block to `easyTipViewDidDismiss` would make more sense than making the user always write this call when programmatically dismissing the view